### ISSUE TITLE
Ubuntu's 18.04 missing -lieee in glibc

### ISSUE THREAD

**Issue by UCano** [ljacobson64/MCNP_CMake#1]
I tried to compile MCNP5.16 in Ubuntu 18.04 with GNU 7.3 compilers in sequential mode. However, the support for libieee appears to be discontinued in glibc 2.27 (default in 18.04 version of Ubuntu).
Therefore I decided to switch to Ubuntu 16.04 and compile with the regular compilers and MPI that worked for me before in regular MCNP compilations. 
Now, cmake cannot find MPI_C although openmpi is properly installed.
Could you point me to the proper OS (I do not care if it is not Ubuntu), compiler versions and MPI versions that have worked for you?

Currently I am just trying to compile version5 1.60, but I will try to compile MCNP6.2 afterwards.
Is MCNP6.2 working with both MPI and OMP with GNU compilers?

Thanks!
(I am currently working in a VM)

**Comment by ljacobson64** [comment]
Hi @UCano,

Sorry for the very late response... it appears I need to turn email notifications on for this repo ;)

My workaround the libieee problem on Ubuntu 18.04 is the following:
```
sudo apt install libieee1284-3-dev
sudo ln -s /usr/lib/x86_64-linux-gnu/libieee1284.so.3 /usr/lib/libieee.so
sudo ln -s /usr/lib/x86_64-linux-gnu/libieee1284.a /usr/lib/libieee.a
```

It might be a better idea to simply remove the `-mieee-fp` flag from the cmake files though. I might do that in the future.

OSes: I've used this repo to build MPI-enabled versions of MCNP with gcc/g++/ifort on Ubuntu 18.04, Debian 8, and Scientific Linux 6.6. I have tested it on older versions of Ubuntu, but not recently.

Compilers: My "main" compilers I use to build MCNP are gcc/g++ 7.3.0 and Intel Fortran (ifort) 18. However I think that versions of gcc/g++ all the way back to at least gcc 4.4.7 should work, and I know that ifort 16 works as well. Gfortran can always be substituted for ifort, but with a performance cost.

Versions of MCNP: All versions of MCNP included in this repo, including MCNP6.2, build successfully with MPI. They all build successfully with OpenMP (except MCNPX because it doesn't support OpenMP in the first place), but only if Intel Fortran is used. I have never been able to compile OpenMP-enabled versions of MCNP using gfortran, and information provided by the MCNP developers at LANL indicates that they cannot do this either.
